src/cmd/kssh/kssh.go

```diff
@@ -35,12 +34,18 @@ func main() {
 		doAction(action, keyPath, remainingArgs)
 		os.Exit(0)
 	}
-	config, err := getConfig(team)
+
+	bot, err := kssh.NewBot()
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		os.Exit(1)
+	}
+	cbot, err := bot.Configure(botName)
```

Comment:
i think a lot of this is more confusing than it needs to be, and this line feels like a good example. first, none of this needs to be up at the level of the stack. pulling client configs so the kssh runner does the right thing can all be hidden down in the call to `provisionNewKey`. that would mitigate another thing i don't love here: the multiple use of `bot`. if i had my druthers, `bot` would only refer to the ca bot, and we'd use some other name (`runner` or `chatter` or `manager` or ...) to refer to the thing that manages chatting over keybase on behalf of the kssh user. bot on this line literally means two completely different things. 

and i have mixed opinions about the spinning out of a new object for the `configured` version of one of these things. i like that it's clearer what's what, and it allows separation of all of the configuration related methods from the actual heavy lifting of ssh work, but is there ever a reason anything might ever want the `unconfigured` version of one of these? calls to `New...` before configuring are already pretty heavy because they're setting up pipes to keybase. why not just do all of the configuration work right there (which i've mentioned above, i think i'd argue we should totally get rid of anyway) and make a cleaner interface to the caller right here?

tl;dr
```golang
func main() {
    caBotName, remainingArgs, action, err := handleArgs(os.Args[1:])
    ...
    provisionNewKey(caBotName, keyPath)
    doAction(action, keyPath, remainingArgs)
}

func provisionNewKey(caBotName, keyPath) {
    applicant := kssh.NewKsshApplicant(caBotName)  // naming things is hard!
    // ^ this object coming out right here would either be "configured"
    ...
    applicant.GetSignedKey()
    // ^ or "configuration" could happen as part of this call. either way, it doesn't really matter up here. 
    ....
}
```

Reply:
i removed the ConfiguredBot abstraction
i renamed the keybase bot package and struct to `ca` and `CertAuthority`, respectively.
i renamed the kssh bot struct to `Requester`.
i moved `New` methods that create a keybase bot API out of the "Action" methods (`serviceAction` and `mainAction`), and wrapped them into helper methods
hope that addresses everything here, lmk